generators/Exercises/Generators/HighScores.cs

```diff
@@ -11,12 +11,26 @@ protected override void UpdateTestMethod(TestMethod testMethod)
             testMethod.ConstructorInputParameters = new[] { "scores" };
 
             testMethod.Input["scores"] = new List<int>(testMethod.Input["scores"]);
-            
+
             if (testMethod.Expected is IEnumerable<int>)
                 testMethod.Expected = new List<int>(testMethod.Expected);
         }
 
-        protected override void UpdateNamespaces(ISet<string> namespaces) 
+        protected override void UpdateTestClass(TestClass testClass)
+            => AddTestMethodForPersonalBest(testClass);
+
+        protected override void UpdateNamespaces(ISet<string> namespaces)
             => namespaces.Add(typeof(List<int>).Namespace);
+
+        private static void AddTestMethodForPersonalBest(TestClass testClass)
+        {
+            testClass.AdditionalMethods.Add(@"[Fact(Skip = ""Remove this Skip property to run this test"")]
+public void Lastest_score_should_not_change_after_calling_personal_best()
```

Comment:
Small typo: 

```suggestion
public void Latest_score_should_not_change_after_calling_personal_best()
```

Reply:
funny one, it was in the original PR too.